core/arch/arm/plat-bcm/conf.mk

```diff
@@ -16,14 +15,22 @@ CFG_TZDRAM_SIZE ?= 0x01000000 # 16MB
 CFG_SHMEM_START ?= ($(CFG_TZDRAM_START) - $(CFG_SHMEM_SIZE))
 CFG_SHMEM_SIZE ?=  0x01000000 # 16MB
 CFG_TEE_RAM_VA_SIZE := 0x400000 # 4MB
-endif
 
 ifeq ($(platform-flavor-armv8),1)
 $(call force,CFG_WITH_ARM_TRUSTED_FW,y)
 $(call force,CFG_PM_STUBS,y)
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 endif
 
+ifeq ($(PLATFORM_FLAVOR),ns3)
+$(call force,CFG_BCM_SOTP,y)
+$(call force,CFG_GPIO, y)
```

Comment:
remove space before `y`
`CFG_GPIO` is not defined yet. Maybe append this change to #2926. Same for `CFG_BCM_GPIO` below.

Reply:
Thanks.
 I can append the driver enabling defines eg: CFG_GPIO ..  to respective driver change  eg:  #2926  for  CFG_GPIO  CFG_BCM_GPIO  And  #2928  for SOTP and HWRNG. I kept it this way to avoid merge conflicts and build dependencies if any.  let me try.  Do you feel platform def also should be split into multiple commits per driver ..
I thought its easier this way.

Reply:
> Thanks.
> I can append the driver enabling defines eg: CFG_GPIO .. to respective driver change eg: #2926 for CFG_GPIO CFG_BCM_GPIO And #2928 for SOTP and HWRNG. I kept it this way to avoid merge conflicts and build dependencies if any. let me try. Do you feel platform def also should be split into multiple commits per driver ..
> I thought its easier this way.

please suggest what else I should do in this PR. Thanks

Reply:
your patch looks good.